eng/tools/spec-gen-sdk-runner/src/types.ts

```diff
@@ -64,8 +64,8 @@ export type SdkName =
  */
 export const SpecGenSdkRequiredSettings: Record<SdkName, boolean> = {
```

Comment:
It might be worth adding a comment that this list is currently only required for management.